### PR TITLE
Docs/mc 15831 set billable org info

### PR DESCRIPTION
**Pull request by HassanHaidar**
#### Changes made
Add documentation for setting billable org info

### CODE REVIEW

**Review by evanluc — APPROVED** [review]
Thanks 🙏

File: source/includes/administration/_organizations.md [hunk 1]
```diff
@@ -818,4 +818,124 @@ Returns an HTTP status code 200, with an empty response body.
...
+   -H "MC-Api-Key: your_api_key" \
+   -H "Content-Type: application/json" \
+   -d "request_body"
+```
+> Request body example:
+
+```json
+{
+   "id": "39f6260d-f0ca-4be9-93c4-46405d471c04",
+   "billableStart": "2015-01-18",
```
Comment (fyrnaga) [inline source/includes/administration/_organizations.md:843]: We need a table to know what can be updated. At the moment, only the billableEnd, billableStart and packageId can be changed.

File: source/includes/administration/_organizations.md [hunk 2]
```diff
@@ -818,4 +818,124 @@ Returns an HTTP status code 200, with an empty response body.
...
+	"pricingPackage": {
+		"id": "58bad1de-1f87-422b-b44f-27ed58889272"
+	}
+}
+```
+
+Required | &nbsp;
+---- | ----
+`id`<br/>*UUID* | The id of the udpated billable organization information.
+`billableStart`<br/>*string*  | The date the organization became billable, in YYYY-MM-DD format.
```
Comment (fyrnaga) [inline source/includes/administration/_organizations.md:853]: missing billableEnd

File: source/includes/administration/_organizations.md [hunk 3]
```diff
@@ -818,4 +818,124 @@ Returns an HTTP status code 200, with an empty response body.
...
+        "id": "de6cf332-14b2-4da8-8522-3dfe12fcd0da"
+      },
+      "name": {
+        "en": "Applied Pricing",
+        "fr": "Applied Pricing"
+      },
+      "currency": "CAD",
+      "id": "92041aa8-7784-45b0-aa85-76376ede784d",
+      "creationDate": "2021-09-13T15:18:38.000Z",
+      "scopeQualifier": "ORG_TREE",
```
Comment (fyrnaga) [inline source/includes/administration/_organizations.md]: Do we have a scope qualifier?

File: source/includes/administration/_organizations.md [hunk 4]
```diff
@@ -818,4 +818,124 @@ Returns an HTTP status code 200, with an empty response body.
...
+
+```json
+{
+  "data": [
+    {
+      "offerType": "30-day-trial",
+      "pricingDefinition": {
+        "id": "c9a396f1-690a-4201-a3bf-b1da37221acc"
+      },
+      "scopeOrganization": {
```
Comment (fyrnaga) [inline source/includes/administration/_organizations.md]: seems like the applied pricing information and not the pricingPackage. There should be no scopeOrganization, no?

File: source/includes/administration/_organizations.md [hunk 5]
```diff
@@ -818,4 +818,124 @@ Returns an HTTP status code 200, with an empty response body.
...
+`id`<br/>*UUID* | The id of the udpated billable organization information.
+`billableStart`<br/>*string*  | The date the organization became billable, in YYYY-MM-DD format.
+`pricingPackage.id`<br/>*UUID* | The id of the pricing package applied on the organization.
+
+<!-------------------- LIST APPLICABLE PRICING PACKAGES TO ORG -------------------->
+### List applicable pricing packages to an organization
+
+`GET /organizations/:organization_id/pricing_packages`
+
+Retrieves a list of applicable pricing packages to an organization.
```
Comment (fyrnaga) [inline source/includes/administration/_organizations.md:865]: Is it really a list of just an object

Comment (fyrnaga) [reply source/includes/administration/_organizations.md:865]: nvm

File: source/includes/administration/_organizations.md (hunk 2 above)
Comment (HassanHaidar) [reply source/includes/administration/_organizations.md:853]: 0a2d885bbed51edfc5051afc4ce786fa623a3d7f

File: source/includes/administration/_organizations.md (hunk 4 above)
Comment (HassanHaidar) [reply source/includes/administration/_organizations.md]: 0a2d885bbed51edfc5051afc4ce786fa623a3d7f

File: source/includes/administration/_organizations.md [hunk 6]
```diff
@@ -818,4 +818,92 @@ Returns an HTTP status code 200, with an empty response body.
...
+      "startDate": "2021-09-22T20:09:32.084Z"
+    }
+  ]
+}
+```
+
+Attributes | &nbsp;
+---- | -----------
+`id`<br/>*UUID* | The UUID of the pricing package.
+`offerTyoe`<br/>*string* | The offer type of the pricing package.
```
Comment (fyrnaga) [inline source/includes/administration/_organizations.md]:
```suggestion
`offerType`<br/>*string* | The offer type of the pricing package.
```

File: source/includes/administration/_organizations.md (hunk 3 above)
Comment (HassanHaidar) [reply source/includes/administration/_organizations.md]: 0a2d885bbed51edfc5051afc4ce786fa623a3d7f

**Review by fyrnaga — APPROVED** [review]

**Pull request merged**